.github/workflows/master.yml

```diff
@@ -17,6 +17,7 @@ env:
   PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
   SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
   TRAVIS_OS_NAME: linux
+  PULUMI_GO_DEP_ROOT: /home/runner/work/pulumi-docker
```

Comment:
No manual changes can be made to this file - this is what yesterdays' lunch and learn was about

Reply:
This change needs to be rolled out as a part of https://github.com/pulumi/pulumi/pull/5264

Rather than waiting for that, I thought I would do this as a one-off to get this issue closed out. When it gets rolled out, there should be no diff for this file.

I've opened an issue to track: https://github.com/pulumi/ci-mgmt/issues/46

Reply:
Let's merge this and I'll roll it out to all the other providers as part of pulumi/ci-mgmt#46